documents/2mer-example

```diff
@@ -0,0 +1,85 @@
+# Conceptual understanding of basecalling
+*This aims to show an example that can aid in simplifying the complexity in DNA sequencing using ONTs nanopore devices. In general, we can view the processing done by the ONT devices as encoding from organic DNA to an electrical signal. However, the encoded output from ONTs devices is impossible for humans to interpret (decode) due to the complex signal generated by the nanopore. Interpreting the signal is referred to as basecalling and is done using artificial intelligence. The topic of this is not to make any statements regarding how to process the signal. The sole purpose is to define tangible examples that can be used to understand the sequencing process itself and underlying obstacles regarding working with the DNA material.*
```

Comment:
```suggestion
*The purpose of this document is to investigate the actual complexity of performing basecalling on data from an ONT MinION device through a small but realistic example.* 

From an outside perspective, the MinION can be seen as a tool for converting organic DNA/RNA material into electrical signals. Basecalling is the process of converting the electrical signals into a sequence of bases, i.e. a sequence over the letters `ACGT`.

The MinION device produces the electrical signal by pulling a piece of RNA through a scanner, called a *nanopore*, and *reading* five bases at a time. A *read* is the resistance measured while sending an electrical current through the five bases inside the nanopore. Basecalling is possible because the resistance varies based on which bases are currently being read.

Based on this description, basecalling might seem trivial, but there are certain complications, which increase the complexity of the problem. What follows is an investigation of these complications through a small and tangible example.
```